ui/pages/login.tsx

```diff
@@ -67,7 +73,8 @@ const Login: NextPage = () => {
         }
         setFlow(data);
       })
-      .catch(handleFlowError("login", setFlow));
+      .catch(handleFlowError("login", setFlow))
+      .catch(redirectToErrorPage);
```

Comment:
If the flow creation fails, then I would expect that there would be no flow_id to set. What will happen if we redirect to the error page with an empty flow_id?

Reply:
We will fall back to a generic message: `An error occurred. Try signing in again.` in case there is no id set or if we can't load the id specified.